<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TripPlanner</class>
 <widget class="QDialog" name="TripPlanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="national_stadiums_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>All National Stadiums</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QPushButton" name="american_stadium_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>All American Stadiums</string>
   </property>
  </widget>
  <widget class="QPushButton" name="custom_stadium_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Custom Stadiums</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QComboBox" name="to_combo_box">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>60</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_button">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
